app/controllers/admin/flags_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Admin::FlagsController < Admin::StaffController
+  include WithServiceHelper
```

Comment:
Let's put this in `AdminController` and `StaffController` base classes so we can use it globally :fire: